src/commands/tasks/sync.ts

```diff
@@ -141,16 +113,20 @@ async function syncSingleTask(sdk:any, config: TaskConfig) {
   }
 
   if (myFunction === undefined) {
-    await createTask(sdk, request);
+    await taskRepository.functions.create(sdk, request);
     console.log(chalk.green('Successfully created task', config.name));
   } else {
     // TODO: Check all fields and only update if they are different
-    const { data: existingFunction } = await sdk.raw.get(`/tasks/v1/functions/${myFunction.name}`);
+    const existingFunction = await taskRepository.functions.findByName(sdk, myFunction.name);
     if (request.runtime === existingFunction.runtime) {
       delete request.runtime;
     }
 
-    await updateTask(sdk, request);
+    const updateResponse = await taskRepository.functions.update(sdk, request);
+    if (updateResponse.data?.affectedRecords) {
```

Comment:
I think the `.data` is incorrect here, as it is already done in the repository function 🤔